src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -231,20 +214,6 @@ const AppWrapper = (props: { children: ReactNode }) => {
   useEffect(() => {
     if (authentication.loggedIn) {
       const handleMessages = async () => {
```

Comment:
OK this useEffect needs some tidying up now. All it will do is sync with KERIA. Pretty cool if it syncs every time we come online, so it can stay in the useEffect. But we can remove `isMessagesHandled` state variable and just call this function `syncWithKeria` instead of `handleMessages`.

Also do we need the `setPauseQueueIncomingRequest` in this specific function? If so, I think we're missing a coordination here in case of the login. Because login unpauses the queue and logout pauses it. Can you check all the options here? e.g. what happens if we are:
- connected to KERIA
- on the login screen, and stay here
- stop KERIA
- start KERIA
- is the incoming queue paused or not?

It should be paused, because we are logged out

Reply:
@iFergal I remember the reason why we have `isMessagesHandled` is just to sync one time. Now we will remove it?

Reply:
We are setting `PauseQueueIncomingRequest` based on the `isOnline` value so I think it should be paused.

Reply:
@Sotatek-HocNguyena Doing it once made sense when it was fetching the notifications from the DB and added them to the queue. Syncing from KERIA we can do more times - I was thinking we could do it whenever we came online.

But, actually, the problem now is that yes I didn't see `if (authentication.loggedIn)` - so this will run whenever we log in and out too.

Re pause incoming request - yes, sorry, looking at it again this was done correctly here. Sorry.

---

One option is 1 useEffect to update `PauseQueueIncomingRequest` based on `loggedIn` and `isOnline`.

And another useEffect to sync with KERIA based just on `isOnline`. (we can sync in the background)

What do you think?

Reply:
Can we just do something like this with just one `useEffect`?
![image](https://github.com/cardano-foundation/cf-identity-wallet/assets/142211701/49c72edc-cead-4046-8435-48718aec5677)

Reply:
@Sotatek-HocNguyena It's better if there's 2 so it's more clear. Also, if we keep logging in and out without going offline, we are constantly checking `isOnline` here when we don't need to.